calculator.js

```diff
@@ -0,0 +1,78 @@
+window.onload = function () {
+  let items = [], sumTotal = 0; 
+  function calculate() {
+    function thousandSeperator(n) {
```

Comment:
nice job, that will enhance the UI by properly seperating large values with commas appropriately

Reply:
thank you